pages/api/mentees/index.js

```diff
@@ -10,16 +10,27 @@ export default async function handler(req, res) {
     return;
   }
 
+  const allFilters = [
+    {$exists: "roles"},
+    {roles: {
+      $includes: "mentee",
+    }},
+  ];
+  if (
+    process.env.NODE_ENV === "development" ||
+    process.env.NEXT_PUBLIC_DEV_LOGIN === "true"
+  ) {
+    allFilters.push({roles: { $includes: "test", }});
+  }
+
```

Comment:
Just curious why we need this? Because all our test users already have all the roles.

Reply:
essentially because at the moment we have real data in the DB, and while playing with accept/reject I wanted to have only test users in the list. Also because the page limit was cutting out the latest mentors. I added a `test` role for our users so they would show up in the list. 

Actually I should filter those users if we do not want them to show up in the real app.

This can be removed once we publish of course.

I will merge this way, and create a follow up ticket. Either to remove the `test` role, either to not consider them in the normal query.

Reply:
there was also something with the `null` values, and I had to add `{$exists: "roles"}` in the filters